renovate.json

```diff
@@ -28,31 +28,31 @@
   "prCreation": "not-pending",
   "packageRules": [
     {
-      "datasources": ["docker"],
+      "matchDatasources": ["docker"],
```

Comment:
We should defer this until the app is updated to new version

Reply:
I updated it today

Reply:
But we need to defer this change until this pr is gone life in the app, otherwise we don't get any new updates because of a wrong renovate config

Reply:
Oh, sorry - I see what you mean. Yes, let's revert this from this PR